clang/test/lab1/kazantsev_evgeny/testDep.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/DeprecatedPlug%pluginext -plugin deprecated-function %s 2>&1 | FileCheck %s --check-prefix=DEPRECATED
+// REQUIRES: plugins
+
+// DEPRECATED: warning: Deprecated function name
+void Deprecated(int a, int b);
+
+// DEPRECATED: warning: Deprecated function name
+void dEpReCaTeD(int c);
+
+// DEPRECATED-NOT: warning: Deprecated function name
+void sqrt();
```

Comment:
Please add a checks for checking class method name.

Reply:
That is what I meant:
```suggestion
class A {
  int method_deprecated();
};
```

Reply:
I've done